documentacion/posintegrado/README.md

```diff
@@ -0,0 +1,275 @@
+segunda pieza, la cual mediante un Wrapper, accede a las funciones disponibles en la librería C.
+
+Adicionalmente, necesitaras tener instalados los drivers correspondientes a tu tarjeta de
+puerto serial o adaptador USB Serial.
+
+<aside class="notice">
+La comunicación con el POS Integrado se realiza mediante puerto serial RS232.
+</aside>
+
+<aside class="notice">
+Tu eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
+</aside>
+
+### LibSerialPort
+
+El SDK depende de [libSerialPort](https://sigrok.org/wiki/Libserialport) para la comunicación serial.
+
+Puedes obtener el código desde el repositorio usando git:
+
+```bash
+git clone git://sigrok.org/libserialport
+```
+
+Para compilar en windows necesitaras lo siguiente:
+
+- [msys2 - mingw-w64](http://www.msys2.org/) Puedes descargarlo siguiendo el link y las instrucciones provistas en su sitio web.
+  - Adicionalmente, necesitaras el toolchain para tu arquitectura:
+    - x86: `pacman -S mingw-w64-i686-toolchain`
+    - x64: `pacman -S mingw-w64-x86_64-toolchain`
+
+<aside class="notice">
+Procura seguir todos los pasos descritos en el sitio de msys2
+</aside>
+
+### Primeros pasos
+
+Para usar el SDK es necesario incluir las siguientes referencias.
+
+<div class="language-simple" data-multiple-language></div>
+
+```csharp
+using Transbank.POS;
+using Transbank.POS.Utils;
+using Transbank.POS.Responses:
+```
+
+```c
+#include "transbank.h"
+#include "transbank_serial_utils.h"
+```
+
+### Listar puertos disponibles
+
+Si los respectivos drivers están instalados, entonces puedes usar la función `ListPorts()` del paquete
+`Transbank.POS.Utils`para identificar los puertos que se encuentren disponibles y seleccionar el que
+corresponda con el puerto donde conectaste el POS Integrado.
+
+<div class="language-simple" data-multiple-language></div>
+
+```csharp
+using Transbank.POS.Utils;
+//...
+List<string> ports = Serial.ListPorts();
+```
+
+```c
+#include "transbank_serial_utils.h"
+//...
+char * ports = list_ports();
+```
+
+### Abrir un puerto Serial
+
+Para abrir un puerto serial y comunicarte con el POS Integrado, necesitaras el nombre del puerto (El cual
+puedes identificar usando la función mencionada en el apartado anterior #Listar puertos disponibles). También
+necesitaras el baudrate al cual esta configurado el puerto serial del POS Integrado (Por defecto es 115200),
+y puedes obtener los distintos valores desde la clase `TbkBaudrates` del paquete `Transbank.POS.Utils`.
+
+Si el puerto no puede ser abierto, se lanzara una exception `TransbankException`.
+
+<div class="language-simple" data-multiple-language></div>
+
+```csharp
+using Transbank.POS;
+using Transbank.POS.Utils;
+//...
+string portName = "COM3";
+POS.Instance.OpenPort(portName, TbkBaudrate.TBK_115200);
+```
+
+```c
+#include "transbank.h"
+#include "transbank_serial_utils.h"
+//...
+char* portName = "COM4";
+int retval = open_port(portName, 115200);
```

Comment:
FWIW, I think in C it's fine to require the baudrate :). I just think that on the wrappers we have to go the extra mile to make the developer experience the best possible